app/routes/routes.py

```diff
@@ -0,0 +1,237 @@
+from flask import Blueprint, make_response, request, jsonify, abort
+from app import db
+from app.models.task import Task
+from datetime import datetime
+from app.models.goal import Goal
+import os, requests
+
+#VALIDATE ID
+def validate_id(class_obj,id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message":f"{id} is an invalid id"}, 400))
+    query_result = class_obj.query.get(id)
+    if not query_result:
+        abort(make_response({"message":f"{id} not found"}, 404))
+
+    return query_result
+
+#CREATE TASK
+task_bp = Blueprint("Task", __name__, url_prefix="/tasks")
+@task_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body:
+        #TODO later wave will need 'or "completed_at" not in request_body', add above
```

Comment:
It's weird to talk about code you didn't write, but this makes sense assuming we want `"completed_at"` to be a required parameter. I think that isn't completely necessary because I think most tasks are going to be sent in uncompleted and while we _can_ force users to supply `"completed_at": null`, if there is no `"completed_at"` key, we can just treat it like we would that case.

Anyway, mainly wanted to point out this little design choice.